PandasWorkshop.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>While this sort of method chaining definitely makes shorter code, I tend to think that in a tutorial we should break it out operation by operation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/orioncohen/metal-bands-by-nation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>